### PR TITLE
power-assert: Update APIs

### PR DESCRIPTION
**Pull request by falsandtru**
Supports https://github.com/power-assert-js/power-assert/pull/104

### CODE REVIEW

File: types/power-assert/index.d.ts [hunk 1]
```diff
@@ -62,6 +62,8 @@ declare namespace assert {
 
     export function ifError(value:any):void;
 
+    export const strict: typeof assert;
```
Comment (paulvanbrenk) [inline types/power-assert/index.d.ts]: Do you need to update the version to 1.5?

**Review by paulvanbrenk — CHANGES_REQUESTED** [review]

File: types/power-assert/index.d.ts (hunk 1 above)
Comment (falsandtru) [reply types/power-assert/index.d.ts]: Yes, we need to update the version to 1.5 to use this API.

Comment (paulvanbrenk) [reply types/power-assert/index.d.ts]: Can you update the version in the header to reflect that. After you've done that I'll merge the change.

Comment (falsandtru) [reply types/power-assert/index.d.ts]: Okay, done.

**Pull request merged**